changelog/738.feature.rst

```diff
@@ -0,0 +1,4 @@
+Add aliases to the ``original_message`` methods.
+- :func:`Interaction.original_response` is aliased to :func:`Interaction.original_message`
+- :func:`Interaction.edit_original_response` is aliased to :func:`Interaction.edit_original_message`
+- :func:`Interaction.delete_original_response` is aliased to :func:`Interaction.delete_original_message`
```

Comment:
small nit, pretty sure these should be the other way around? i.e. `[thing] is aliased to [target]`

Reply:
This was intentional because we aren't deprecating original_message in this PR--only removing it from documentation.